### PR TITLE
docs: document docs commit check

### LINKED ISSUE #1796: docs: add details of PR check for docs commit in contribution guide
-------------------

### Description
Recently, we added bot-check that ensures that each PR contains relevant documentation changes as well. 

This will be a frequently faced scenario by developers when they try to push PRs. Guidelines for the same will be needed. 

-------------------

### Prepare

- [x] Read contribution guidelines
- [x] Read license information

### PR DESCRIPTION
**Pull request by ossdhaval**
### Prepare

- [x] Read [PR guidelines](https://github.com/JanssenProject/jans/blob/main/docs/CONTRIBUTING.md#prs)
- [x] Read [license information](https://github.com/JanssenProject/jans/blob/main/LICENSE)

-------------------

### Description
Added guidelines around documentation commit in CONTRIBUTING.md

#### Target issue
 closes #1796

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -208,7 +208,22 @@ To run tests locally before pushing your code, refer to [TESTING] guide.
 
 ## Document
 
-PR should include changes in relevent documentation along with code changes.
+- PR should include changes in relevant documentation along with code changes.  PR is checked by bot to have either 
+of the following
```
Comment (moabu) [inline docs/CONTRIBUTING.md]: either one of the following :

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -208,7 +208,22 @@ To run tests locally before pushing your code, refer to [TESTING] guide.
...
+  - A commit that follows [commit guidelines](#commits) with `docs:` message 
+  - Changes in artifacts under `jans/docs`
+  
+- If PR does not need any documentation changes, then developer need to acknowledge that too in one of two ways:
```
Comment (moabu) [inline docs/CONTRIBUTING.md]: If PR does not need any documentation changes, then **the** developer need**s** to acknowledge that ~~too~~ in one of two ways:

**Review by moabu — CHANGES_REQUESTED** [review]

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (ossdhaval) [reply docs/CONTRIBUTING.md]: Fixed with [cf97cff](https://github.com/JanssenProject/jans/pull/1797/commits/cf97cffb27c8e76fe559192c3456f372f6eaa20d)

File: docs/CONTRIBUTING.md (hunk 2 above)
Comment (ossdhaval) [reply docs/CONTRIBUTING.md]: Fixed with [cf97cff](https://github.com/JanssenProject/jans/pull/1797/commits/cf97cffb27c8e76fe559192c3456f372f6eaa20d)

**Review by moabu — APPROVED** [review]

**Pull request merged**